src/components/modals/login-modal/forgot-password-content.less

```diff
@@ -0,0 +1,72 @@
+.forgot-password-title-text {
+  position: absolute;
+  width: 306px;
+  height: 52px;
+  top: 112px;
+  left: 161px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 28px;
+  line-height: 140.62%;
+  align-items: center;
+  text-align: center;
+  color: #000000;
+}
+
+.subtitle-text {
+  position: absolute;
+  width: 451px;
+  height: 34px;
+  top: 164px;
+  left: 97px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 18px;
+  line-height: 140.62%;
+  align-items: center;
+  text-align: center;
+  color: #000000;
+}
+
+.email-label {
+  position: absolute;
+  width: 132.88px;
+  height: 18px;
+  top: 234px;
+  left: 108px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 163.6%;
+  display: flex;
+  align-items: center;
+  color: #000000;
+}
+
+.forgot-password-email-field {
+  position: absolute;
+  width: 410px;
+  height: 48px;
+  top: 258px;
+  left: 108px;
+  background: #e5e5e5;
+  border-radius: 5px;
+}
+
+.input-style {
+  margin-bottom: 0px;
+  margin-top: 0px;
+}
+
+.ant-btn.ant-btn-primary.send-link-button {
```

Comment:
We want to avoid overwriting AntD classes since they are global styles that may affect other components! If possible can you change this to not use AntD classes: i.e. styled-components or just applying a custom class?

Reply:
If we set the global primary color, would that propagate and set the button color?